### PR TITLE
Move login form to a new login screen.

### PR DESCRIPTION
**Pull request by rdwebdesign**
**Please make sure you**
 1. Base your code and PRs against the repositories developmental branch.
 2. [Sign Off](https://docs.pi-hole.net/guides/github/how-to-signoff/) all commits as we enforce the [DCO](https://docs.pi-hole.net/guides/github/dco/) for all contributions

---
- **What does this PR aim to accomplish?:**

This PR creates a new login page, with its own layout.

newloginpage

- **How does this PR accomplish the above?:**

Removing the old login page and creating a new one.

- **What documentation changes (if any) are needed to support this PR?:**

Maybe a few screenshots.

---
**By submitting this pull request, I confirm the following:** 

1. I have read and understood the [contributors guide](https://docs.pi-hole.net/guides/github/contributing/), as well as this entire template. I understand which branch to base my commits and Pull Requests against. 
2. I have commented my proposed changes within the code and I have tested my changes.
3. I am willing to help maintain this change if there are issues with it later.
4. It is compatible with the [EUPL 1.2 license](https://opensource.org/licenses/EUPL-1.1)
5. I have squashed any insignificant commits. ([`git rebase`](http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html))

---
- [x] I have read the above and my PR is ready for review. _Check this box to confirm_

### CODE REVIEW

**Comment by DL6ER** [comment]
Why are you introducing a new file `common_header.php` in addition to the already existing `header.php`? This naming convention seems somewhat un-intuitive.

File: scripts/pi-hole/php/password.php [hunk 1]
```diff
@@ -9,114 +9,100 @@
...
-if(isset($setupVars['WEBPASSWORD']))
-{
+if (isset($setupVars['WEBPASSWORD'])) {
     $pwhash = $setupVars['WEBPASSWORD'];
-}
-else
-{
+} else {
     $pwhash = "";
 }
```
Comment (DL6ER) [inline scripts/pi-hole/php/password.php]: This looks like an unnecessary code-style change. Even more of this happens below. If we ever were to track down changes "when did what happen" in the future, such changes without any really change always make lives more difficult without any apparent gain.

Comment (PromoFaux) [reply scripts/pi-hole/php/password.php]: https://javascript.info/coding-style#curly-braces

Comment (PromoFaux) [reply scripts/pi-hole/php/password.php:28]: But really, styling changes should come in a separate PR/commit I suppose (though toggling "hide whitespace" makes it easer to review)

Comment (PromoFaux) [reply scripts/pi-hole/php/password.php]: But really, styling changes should come in a separate PR/commit I suppose

Comment (rdwebdesign) [reply scripts/pi-hole/php/password.php]: Actually, the link you posted is about javascript style.

I understand the code style is a personal preference, but we need to choose and follow a single standard.
Currently there is no style for PHP files (or the style is a complete mess). I want to fix this using the same style to all PHP files.

I can suggest one of the known PHP styles ([PSR-2](https://www.php-fig.org/psr/psr-2/), [PSR-12](https://www.php-fig.org/psr/psr-12/)).

If you like, I can delay this PR and format all PHP files in a new PR.

**Comment by rdwebdesign** [comment]
> Why are you introducing a new file common_header.php in addition to the already existing header.php? This naming convention seems somewhat un-intuitive.

Both `header.php` and the new `login.php` use `common_header.php`.
All other pages use `header.php`.

I think we can change the code to remove this extra include, but I will need to change every page currently including `header.php`.

File: scripts/pi-hole/php/password.php (hunk 1 above)
Comment (yubiuser) [reply scripts/pi-hole/php/password.php]: For Vs code there are (multiple) php-code style fixers (e.g. https://github.com/junstyle/vscode-php-cs-fixer).

We could also consider adding an action to check the code style: https://github.com/OskarStark/php-cs-fixer-ga which uses https://github.com/FriendsOfPHP/PHP-CS-Fixer internally.

Comment (rdwebdesign) [reply scripts/pi-hole/php/password.php]: Yes, like we already use for other languages.

Comment (dschaper) [reply scripts/pi-hole/php/password.php]: There's some PHP modules for `prettier` too. I have most of my linting via `prettier`.

Comment (rdwebdesign) [reply scripts/pi-hole/php/password.php]: I read somewhere prettier-PHP is not so good for files mixing HTML and PHP (our case), but we can try it.

Comment (yubiuser) [reply scripts/pi-hole/php/password.php]: Yes, fixing mixed html/php is considered unstable: https://github.com/prettier/plugin-php

I don't see a way to specify `PSR-12` or `PSR-2` directly with this plugin.

Comment (rdwebdesign) [reply scripts/pi-hole/php/password.php]: `prettier` doesn't allow select many options. It uses what `prettier` developers decided.

**Comment by rdwebdesign** [comment]
I will finish this PR after we resolve the "style" action conversation.

File: scripts/pi-hole/php/password.php (hunk 1 above)
Comment (DL6ER) [reply scripts/pi-hole/php/password.php]: `PSR-2` has been superseded by `PSR-12` according to [the specifications](https://www.php-fig.org/psr/psr-12/) you liked for `PSR-12`:

> This specification extends, ***expands and replaces [PSR-2](https://www.php-fig.org/psr/psr-2/)***, the coding style guide and requires adherence to [PSR-1](https://www.php-fig.org/psr/psr-1/), the basic coding standard.

Comment (rdwebdesign) [reply scripts/pi-hole/php/password.php]: Yes, it includes PSR-2 + PSR-1 rules, but most new rules in `PSR-12` are related to classes (which we don't use in our code) and "new functionalities" like type declarations (`declare(strict_types=1)`).

**Note:**
Not really _new_ functionality, but from PHP7.4+.

We still support PHP5.6 and we don't use classes, so I think we can use PRS-2 style without any problem.

Comment (DL6ER) [reply scripts/pi-hole/php/password.php]: @rdwebdesign I pushed a new branch using the linter suggested by @yubiuser above: [tweak/php-cs-fixer](https://github.com/pi-hole/AdminLTE/tree/tweak/php-cs-fixer)

I used `php-cs-fixer` locally to fix all files and will open a PR now with some further details.

Comment (DL6ER) [reply scripts/pi-hole/php/password.php]: https://github.com/pi-hole/AdminLTE/pull/2288

**Comment by rdwebdesign** [comment]
Superseded by https://github.com/pi-hole/AdminLTE/pull/2290

**Pull request closed without merging**